docs/getting-started/server/guide.md

```diff
@@ -328,8 +337,8 @@ repository.
 
     </community>
 
-3.  Once you have your `secrets.json` complete, run this command to add the secrets to each
-    Bitwarden server project:
+3.  Once you have your `secrets.json` complete, run the below command to add the secrets to each
+    Bitwarden server project. You may have to `brew install dotnet@6` first.
```

Comment:
Is there a specific error you saw if you do not have dotnet 6 installed 👀 That may be good to detail here if you think it would help!